code/parse/parse_manager.h

```diff
@@ -0,0 +1,60 @@
+#include "globalincs/systemvars.h"
+#include "globalincs/pstypes.h"
+
+enum class Parse_Input_Types : std::uint8_t
```

Comment:
I don't see a reason to have a type for this enum since it is not used for serialization anyway.